static/sidebar/js/modal/index.js

```diff
@@ -54,6 +54,10 @@
         var holder = Utils.getByClass('tvp-products-holder');
```

Comment:
Please remove your changes related to ratings.
We will include this in a separate branch.